src/ONYXKEYS.ts

```diff
@@ -106,27 +103,52 @@ const ONYXKEYS = {
     STASHED_SESSION: 'stashedSession',
     BETAS: 'betas',
 
-    /** NVP keys
+    /** NVP keys */
+
+    /** Boolean flag only true when first set */
+    NVP_IS_FIRST_TIME_NEW_EXPENSIFY_USER: 'nvp_isFirstTimeNewExpensifyUser',
+
     /** Contains the user preference for the LHN priority mode */
     NVP_PRIORITY_MODE: 'nvp_priorityMode',
 
     /** Contains the users's block expiration (if they have one) */
-    NVP_BLOCKED_FROM_CONCIERGE: 'private_blockedFromConcierge',
+    NVP_BLOCKED_FROM_CONCIERGE: 'nvp_private_blockedFromConcierge',
 
     /** A unique identifier that each user has that's used to send notifications */
-    NVP_PRIVATE_PUSH_NOTIFICATION_ID: 'private_pushNotificationID',
+    NVP_PRIVATE_PUSH_NOTIFICATION_ID: 'nvp_private_pushNotificationID',
 
     /** The NVP with the last payment method used per policy */
-    NVP_LAST_PAYMENT_METHOD: 'nvp_lastPaymentMethod',
+    NVP_LAST_PAYMENT_METHOD: 'nvp_private_lastPaymentMethod',
 
     /** This NVP holds to most recent waypoints that a person has used when creating a distance request */
     NVP_RECENT_WAYPOINTS: 'expensify_recentWaypoints',
 
     /** This NVP will be `true` if the user has ever dismissed the engagement modal on either OldDot or NewDot. If it becomes true it should stay true forever. */
-    NVP_HAS_DISMISSED_IDLE_PANEL: 'hasDismissedIdlePanel',
+    NVP_HAS_DISMISSED_IDLE_PANEL: 'nvp_hasDismissedIdlePanel',
 
     /** This NVP contains the choice that the user made on the engagement modal */
-    NVP_INTRO_SELECTED: 'introSelected',
+    NVP_INTRO_SELECTED: 'nvp_introSelected',
+
+    /** This NVP contains the active policyID */
+    NVP_ACTIVE_POLICY_ID: 'nvp_expensify_activePolicyID',
+
+    /** This NVP contains the referral banners the user dismissed */
+    NVP_DISMISSED_REFERRAL_BANNERS: 'nvp_dismissedReferralBanners',
+
+    /** Indicates which locale should be used */
+    NVP_PREFERRED_LOCALE: 'nvp_preferredLocale',
+
+    /** Whether the user has tried focus mode yet */
+    NVP_TRY_FOCUS_MODE: 'nvp_tryFocusMode',
+
+    /** Whether the user has been shown the hold educational interstitial yet */
+    NVP_HOLD_USE_EXPLAINED: 'holdUseExplained',
```

Comment:
should this one be `nvp_holdUseExplained`

Reply:
Can we delete it?

Reply:
I am not sure honestly, this seems broken and kind of outside of the scope of this PR to fix it or remove it./